playground/frontend/lib/services/analytics/events/constants.dart

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class PlaygroundAnalyticsEvents {
+  static const loaded = 'loaded';
+  static const newExample = 'new_example';
+  static const shareableCopied = 'shareable_copied';
+  static const shareCodeClicked = 'share_code_clicked';
+  static const shortcutsClicked = 'shortcuts_clicked';
+  static const snippetSelected = 'snippet_selected';
+}
+
+class PlaygroundEventParams {
+  static const shareFormat = 'shareFormat';
```

Comment:
We namespace the param name constants with classes like this one. In `playground_components` there is a twin class with more params: https://github.com/akvelon/beam/blob/fb34d74a72822c1cb4d251bc96065c5c69a6894c/playground/frontend/playground_components/lib/src/services/analytics/events/constants.dart#L33

In this case we only have one but still want to namespace it for symmetry. This will pay if we add more params to the playground-app-specific events.